src/create-preview-links.ts

```diff
@@ -28,7 +37,11 @@ export const COMMENT_BLOCK_START = '### Preview changes';
  * @param {string} branch - The branch where the theme changes are located.
```

Comment:
It looks like we need to update the inline docs here to include the new `repo` param.

Reply:
Addressed in https://github.com/vcanales/action-wp-playground-pr-preview/pull/24/commits/225fc4c2e147e92ca2c82412889ef2c3286ab6e0